src/main/java/domain/Car.java

```diff
@@ -0,0 +1,30 @@
+package domain;
+
+public class Car {
+
+    private String carName;
+    private int position;
+
+    public Car(String carName) {
+        this.carName = carName;
+        this.position = 0;
+    }
```

Comment:
정말 좋은데요? Car 객체를 만들 때는 보통 **이름만 필요**할 것 같아요!

제 생각엔 0도 충분한 의미를 가진 숫자라서, 이 생성자로 생성되는 객체는 꼭 `초기 상태`여야 할 것 같아요!

그렇다면 그 상황에만 사용하도록 [정적 팩터리 메서드](https://tecoble.techcourse.co.kr/post/2020-05-26-static-factory-method/)를 사용해보는 것은 어떨까요?

Reply:
생성자의 필요한 파라미터가 여러개일 때는 다음과 같이(생성자 체이닝) 표현해보는 것은 어떨까요? (중요도 낮음)
```suggestion
    public Car(String carName) {
        this(carName, 0);
    }
```

Reply:
<정적 팩터리 메서드란 객체 생성에 사용되는 메서드로, 주로 클래스의 생성자 대신 static 메서드로 객체를 생성하는 방식입니다.> 정적 팩터리 메서드는 생성자와 달리 이름을 가질 수 있기 때문에 객체를 생성하는 의도를 더 명확히 전달할 수 있는 장점이 있겠네요! 좋은 의견 감사합니다.